### PR TITLE
Airbus pattern use

### LINKED ISSUE #614: Airbus + Citi as Known Instances of InnerSource License
They presented about it at the InnerSource Summit 2023, we can use the recording as reference.

### PR DESCRIPTION
**Pull request by sicot-f**
We are adding Airbus' use of Innersource patterns into various known instances.

### CODE REVIEW

File: patterns/2-structured/innersource-license.md [hunk 1]
```diff
@@ -66,6 +66,8 @@ The mentioned collaboration challenges include:
 
 It is worth mentioning that so far the software shared under this InnerSource license is mostly tooling, infrastructure, and tools lower in the stack.
 
+Airbus created ad hoc InnerSource licenses to enable InnerSource way of working within a large part of the group.
```
Comment (spier) [inline patterns/2-structured/innersource-license.md:69]: What do you mean by "ad hoc" in this context?
Do you have a template that people should use to create their own licenses? Or do you prescribe a single one? Or something else? :)

Also can you share if you modelled your licenses after any open source license that is out there, and why?

File: patterns/2-structured/repository-activity-score.md [hunk 2]
```diff
@@ -115,6 +115,7 @@ The repository activity score is a simple calculation based on the GitHub API. I
 ## Known Instances
 
 * Used in SAP's InnerSource project portal to define the default order of the InnerSource projects. It was first created in July 2020 and is fine-tuned and updated frequently ever since. When proposed to the InnerSource Commons in July 2020, this pattern emerged. Also see [Michael Graf & Harish B (SAP) at ISC.S11 - The Unexpected Path of Applying InnerSource Patterns](https://www.youtube.com/watch?v=6r9QOw9dcQo&list=PLCH-i0B0otNQZQt_QzGR9Il_kE4C6cQRy&index=6).
+* Airbus took a lot of inspiration from this pattern to create an "InnerSource score" that combines the activity score together with checks from the [Standard Base Documentation](./project-setup/base-documentation.md) and the [InnerSource License](./innersource-license.md).
```
Comment (spier) [inline patterns/2-structured/repository-activity-score.md:118]: ❤️

File: patterns/2-structured/dedicated-community-leader.md [hunk 3]
```diff
@@ -59,6 +59,7 @@ Having excellent and dedicated community leaders is a precondition for the succe
 ## Known Instances
 
 * _BIOS at Robert Bosch GmbH_. Note that InnerSource at Bosch was, for the majority, aimed at increasing innovation and to a large degree dealt with internal facing products. This pattern is currently not used at Bosch for lack of funding.
+* _Airbus_. A data scientist wanted to improve the collaboration with peers in the group and found: i) many developers (beyond data science) wanted that too and were happy someone was taking care of the issue, and ii) support from line manager and middle management to eventually act as the _de facto_ community leader, on top of his regular line of duty.
```
Comment (spier) [inline patterns/2-structured/dedicated-community-leader.md:62]: Please ignore the spell checking issues reported here.
I will figure out how to fix this in our automation.

**Review by spier — COMMENTED** [review]
Thank you for sharing these experiences with us @sicot-f !

I left one question in this review that could help us make the content even better.
Otherwise this already looks great.

File: patterns/2-structured/innersource-license.md (hunk 1 above)
Comment (sicot-f) [reply patterns/2-structured/innersource-license.md:69]: By "ad hoc" I mean we created them from scratch (not based on open-source licenses) for the sole sake of Airbus InnerSource program. Our IP team was used to commercial licenses while developers were more familiar with open-source ones. We iterated many times to find common grounds and also comply with the transfer pricing directive. I'll see with my colleagues if I can share more. Keep you posted.

Comment (spier) [reply patterns/2-structured/innersource-license.md:69]: That would be awesome!

**Comment by spier** [comment]
@sicot-f given the feedback you shared I am already going ahead and merging what we have here.

If you have more info that you want to share publicly, please open a new PR.

Who knows, maybe you can even share key parts of your license as part of the InnerSource License pattern, just like DB Systel did. Of course that is more effort, in terms of getting approval etc.

**Comment by spier** [comment]
This is now live, and you can find all patterns containing "Airbus" like this:
https://patterns.innersourcecommons.org/p/innersource-license?q=airbus

Thank you again for your contribution @sicot-f !

**Pull request merged**